packages/docs/src/pages/index.md

```diff
@@ -14,7 +14,11 @@ The design system is a set of open source design and front-end development resou
 
 It is currently being applied to [HealthCare.gov](https://www.healthcare.gov/). It is open source and freely available to use by anyone.
 
-<a href="{{root}}/startup/installation/" class="ds-c-button ds-c-button--primary ds-c-button--big">Learn how to get started</a>
```

Comment:
Added a section for developers and designers to the homepage. 👍

Reply:
I like this idea, but again, we've had cases where non-devs need the dev stuff, and non-designers need the design stuff. These labels can be off-putting, and I don't think we actually need to separate the content by role.